bulk/management/commands/data_quality.py

```diff
@@ -0,0 +1,343 @@
+            total_versions_per_bill.append(bill.versions.count())
+
+        average_sponsors_per_bill = round(mean(total_sponsorships_per_bill))
+        average_actions_per_bill = round(mean(total_actions_per_bill))
+        average_votes_per_bill = round(mean(total_votes_per_bill))
+        average_documents_per_bill = round(mean(total_documents_per_bill))
+        average_versions_per_bill = round(mean(total_versions_per_bill))
+
+        min_sponsors_per_bill = round(min(total_actions_per_bill))
+        max_sponsors_per_bill = round(max(total_actions_per_bill))
+
+        min_actions_per_bill = round(min(total_actions_per_bill))
+        max_actions_per_bill = round(max(total_actions_per_bill))
+
+        min_votes_per_bill = round(min(total_votes_per_bill))
+        max_votes_per_bill = round(max(total_votes_per_bill))
+
+        min_documents_per_bill = round(min(total_documents_per_bill))
+        max_documents_per_bill = round(max(total_documents_per_bill))
+
+        min_versions_per_bill = round(min(total_versions_per_bill))
+        max_versions_per_bill = round(max(total_versions_per_bill))
+
+        average_num_data[chamber_name].append({
+            "chamber": chamber_name,
+            "average_sponsors_per_bill": average_sponsors_per_bill,
+            "min_sponsors_per_bill": min_sponsors_per_bill,
+            "max_sponsors_per_bill": max_sponsors_per_bill,
+
+            "average_actions_per_bill": average_actions_per_bill,
+            "min_actions_per_bill": min_actions_per_bill,
+            "max_actions_per_bill": max_actions_per_bill,
+
+            "average_votes_per_bill": average_votes_per_bill,
+            "min_votes_per_bill": min_votes_per_bill,
+            "max_votes_per_bill": max_votes_per_bill,
+
+            "average_documents_per_bill": average_documents_per_bill,
+            "min_documents_per_bill": min_documents_per_bill,
+            "max_documents_per_bill": max_documents_per_bill,
+
+            "average_versions_per_bill": average_versions_per_bill,
+            "min_versions_per_bill": min_versions_per_bill,
+            "max_versions_per_bill": max_versions_per_bill}
+        )
+    return average_num_data
+
+
+def no_sources(bills, chambers):
+    no_sources_data = defaultdict(list)
+    for chamber in chambers:
+        chamber_name = chamber.name.lower()
+        total_bills_no_sources = bills.filter(from_organization=chamber, sources=None).count()
+        total_votes_no_sources = bills.filter(from_organization=chamber, votes__sources=None).count()
+        no_sources_data[chamber_name].append({
+            "chamber": chamber_name,
+            "total_bills_no_sources": total_bills_no_sources,
+            "total_votes_no_sources": total_votes_no_sources}
+        )
+    return no_sources_data
+
+
+def bill_subjects(bills, chambers):
+    bill_subjects_data = defaultdict(list)
+    for chamber in chambers:
+        chamber_name = chamber.name.lower()
```

Comment:
chamber.classification is going to be more reliable here

Reply:
Now using chamber.classification